tests/tests.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+import json
+
+from openfisca_web_api.app import create_app
+
+tester = create_app('openfisca_dummy_country').test_client()
+
+
+class ParametersRoute(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.response = tester.get('/parameters')
+
+    def test_return_code(self):
+        self.assertEqual(self.response.status_code, 200)
+
+    def test_headers(self):
+        headers = self.response.headers
+        self.assertEqual(headers.get('Country-Package'), 'openfisca-dummy-country')
+        self.assertEqual(headers.get('Country-Package-Version'), '0.1.1')
+
+    def test_item_content(self):
+        parameters = json.loads(self.response.data)
+        self.assertEqual(
+            parameters[u'impot.taux'],
+            { u'description': u'taux d\'impôt sur les salaires' }
+            )
+
+
+class ParameterRoute(TestCase):
+
+    def test_error_code_non_existing_parameter(self):
+        response = tester.get('/parameter/non.existing.parameter')
+        self.assertEqual(response.status_code, 404)
+
+    def test_return_code_existing_parameter(self):
+        response = tester.get('/parameter/impot.taux')
+        self.assertEqual(response.status_code, 200)
+
```

Comment:
> All these requests return a `404`.

That's not the point. In software testing, you want to do [equivalence partitioning](https://en.wikipedia.org/wiki/Equivalence_partitioning) of the _input_ data, not the output.

> What would these tests bring us ?

Understanding of the limits, and future-proofing. The fact that they yield a 404 vs a 500 is informative, and this guarantee should be checked.

You could also enrich the dummy country package with the expected unicode field, if you want to support it.